Realm/RLMSyncSessionRefreshHandle.mm

```diff
@@ -30,16 +30,28 @@
 
 using namespace realm;
 
+namespace {
+void runBlockForUserContext(const std::weak_ptr<SyncUser>& user,
```

Comment:
This is only ever used to call `unregister_refresh_handle`, so it should probably just encapsulate that call at this point.